packages/backend/checkcaptcha/utils/getReCaptchaKeys.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Retrieve the private keys needed for Ethereums settlement and interactions
+ * 
+ * @param {SecretsManager} secretsManager Instance of AWS SDK Secrets Manager
+ * @returns {String, String} Returns Google Project API Key, Google Cloud project ID, ReCaptcha v3 Enterprise key, ReCaptcha v3 Action and ReCaptcha Score Threshold
+ */
+async function getReCaptchaKeys(secretsManager) {
+  let googleApiKey, googleProjectId, reCaptchaKey, reCaptchaAction, scoreThreshold;
+
+  let googleApiKeyPromise = secretsManager.getSecretValue({ SecretId: 'nouns/GoogleApiKey' }, (_, data) => {
+    googleApiKey = data.SecretString;
+  }).promise();
+
+  let googleProjectIdPromise = secretsManager.getSecretValue({ SecretId: 'nouns/GoogleProjectId' }, (_, data) => {
+    googleProjectId = data.SecretString;
+  }).promise();
+
+  let reCaptchaKeyPromise = secretsManager.getSecretValue({ SecretId: 'nouns/ReCaptchaKey' }, (_, data) => {
+    reCaptchaKey = data.SecretString;
+  }).promise();
+
+  let reCaptchaActionPromise = secretsManager.getSecretValue({ SecretId: 'nouns/ReCaptchaAction' }, (_, data) => {
+    reCaptchaAction = data.SecretString;
+  }).promise();
+
+  let scoreThresholdPromise = secretsManager.getSecretValue({ SecretId: 'nouns/ReCaptchaThreshold' }, (_, data) => {
```

Comment:
Per above, let's just move this into the `app..js`. No need for this to be stored in KMS.

Reply:
It's now provided as ENV variable to lambda function